Makefile

```diff
@@ -25,13 +25,13 @@ build: clean ## build binaries by default
 	@echo "build sealer and seautil bin"
 	hack/build.sh
 
-linux: clean ## build binaries for linux
-	@echo "build sealer and seautil bin for linux"
-	GOOS=linux GOARCH=amd64 hack/build.sh $(GitTag)
+linux: clean ## Build binaries for Linux
+	@echo "Building sealer and seautil binaries for Linux (amd64)"
+	GOOS=linux GOARCH=amd64 hack/build.sh $(GIT_TAG)
```

Comment:
could we use local go env `GOARCH` to replace it ? make it as a variable

Reply:
`GIT_TAG` is nil?

Reply:
I didn't change it, and I wonder why, but it didn't affect my make.I incorporate it into later tasks

Reply:
maybe this variable is deleted, you can optimize it

Reply:
yep